### ISSUE TITLE
image format issues

### ISSUE THREAD

**Issue by delarosatrevin** [I2PC/scipion#493]
Just to summary the issues that we currently have with some image formats.
This issues could be further split into several ones.

* We can not read DM4 (maybe take a look at EMAN2 code)
* We can not read new IMAGIC format (we could get some sample files and also check EMAN2 code)
* We do not follow the MRC2014 convention (probably check with @rmarabini )

**Comment by azazellochg** [comment]
Imagic files created in new imagic, can be imported into scipion without errors (via e2ih.py).  Should I check some special cases?

**Comment by delarosatrevin** [comment]
Yes, as I mentioned before, I did the trick with e2ih.py to handle dm4 and imagic format. 
But I want to keep this issue to implement the support natively in Xmipp.

**Comment by azazellochg** [comment]
@delarosatrevin :+1: 
links to eman2 code: [dm4](https://github.com/cryoem/eman2/blob/master/libEM/dm4io.cpp), [old imagic](https://github.com/cryoem/eman2/blob/master/libEM/imagicio.cpp), [new imagic](https://github.com/cryoem/eman2/blob/master/libEM/imagicio2.cpp)
mrc2014: [format](http://www.ccpem.ac.uk/mrc_format/mrc_format.php), [validation](http://emportal.nysbc.org/mrc2014/)
